### PR TITLE
Add a (information) validity attribute to cards

### PR DESCRIPTION
**Pull request by khcr**
The attribute `validity` can have values `correct`, `outdated` or `disabled`.
It is updated accordingly in the cards check task and used in the card's edit form.

### CODE REVIEW

File: app/models/card.rb [hunk 1]
```diff
@@ -9,6 +9,7 @@ class Card < ApplicationRecord
 
   enum card_type: [:youth, :adult, :activist, :organization, :network, :training]
   enum status: [:pending, :online, :incomplete, :change]
+  enum validity: [:correct, :outdated, :disabled]
```
Comment (nkcr) [inline app/models/card.rb:12]: I would use: `[:maintained, :abandoned, :disabled]`

File: lib/tasks/cards_update_check.rake [hunk 2]
```diff
@@ -32,13 +32,14 @@ namespace :cards_update do
...
         # => To be implemented in model
-        #card.deactivated = true
-        #card.save
+        card.update_attribute(:validity, :disabled)
 
         Admin::CardMailer.deactivated(card).deliver_now
       end
 
       if step != -1
+        card.update_attribute(:validity, :outdated) if step < 4
```
Comment (nkcr) [inline lib/tasks/cards_update_check.rake:41]: If it makes sense:

```suggestion
        card.update_attribute(:validity, :outdated) if step < 4 && !@card.outdated
```

**Review by nkcr — COMMENTED** [review]
Much better 👍

File: app/models/card.rb (hunk 1 above)
Comment (nkcr) [reply app/models/card.rb:12]: Or `[:maintained, :outdated, :disabled]`...?

Comment (khcr) [reply app/models/card.rb:12]: Or [:maintained, :solicited, :disabled], or [:maintained, :requested, :disabled] ?

**Review by nkcr — APPROVED** [review]

**Pull request merged**